vcd_cli/catalog.py

```diff
@@ -111,6 +114,31 @@ def info(ctx, catalog_name, item_name):
     except Exception as e:
         stderr(e, ctx)
 
+@catalog.command(short_help='rename catalog and/or change catalog description')
+@click.pass_context
+@click.argument('catalog-name',
+                metavar='<catalog-name>',
+                required=True)
+@click.option('-n',
+              '--new_catalog_name',
+              required=False,
+              default=None,
+              metavar='[new-catalog-name]')
+@click.option('-d',
+              '--description',
+              required=False,
+              default=None,
+              metavar='[description]')
+def update(ctx, catalog_name, new_catalog_name, description):
```

Comment:
Can we make sure this work for all the 3 senarios?

1. Login as System. and update the catalog
2. Login as System , switch to 'non-system-org' and update catalog
3. Login as 'non-system-org' and update catalog.

Reply:
Not sure what you are suggesting. Are you saying that I should have these as test cases in the unit test? Are there any code changes required?

Reply:
I was asking if all the above 3 cases were manually tested.

Reply:
I see. No they were not, but I will manually test them.

Reply:
What is the python code for switching from System to a non-system-org?

Reply:
Can System have catalogs, or can only non-system orgs can have catalogs?

Reply:
I am unable to create a catalog in System because it does not have any storage resources associated with it, and there does not seem to be a way to add storage resources to System, only to non-system-orgs. Can you please advise? Thanks.

Reply:
Yes. System will not catalogs. You can test the other 2 cases.

Reply:
#3 has been done. #2 has also been done by doing this from the command line:
vcd org use STFTestOrg
And then running the unittest. If you mean I should change the unittest to accomplish #2, let me know. Thanks.

Reply:
I am not sure if we can write unitttest by using different orgs. You can leave the unnittest as it is.

Reply:
OK, then #2 and #3 have been done.